client/src/pages/cart.jsx

```diff
@@ -0,0 +1,189 @@
+              {item.qty} x {item.price.toFixed(2)} €
+            </CartItemQuantityAndPrice>
+          </CartItem>
+          <CartItemSeparation />
+        </CartListWrapper>
+      ))}
+      {cartItems.length !== 0 && (
+        <>
+          <CoffeePriceBox>
+            <CoffeePriceText>Coffee Price</CoffeePriceText>
+            <CoffeePrice>{itemsPrice.toFixed(2)} €</CoffeePrice>
+          </CoffeePriceBox>
+          <ShippingPriceBox>
+            <ShippingPriceText>Shipping Price</ShippingPriceText>
+            <ShippingPrice>{shippingPrice.toFixed(2)} €</ShippingPrice>
+          </ShippingPriceBox>
+          <TotalPriceBox>
+            <div>Total Price</div>
+            <div>{totalPrice.toFixed(2)} €</div>
+          </TotalPriceBox>
+        </>
+      )}
+      <NavBar />
+    </>
+  );
+}
+
+export default Cart;
+
+const CartTitle = styled.h2`
+  font-size: 1.3rem;
+  margin-bottom: 1rem;
+  margin-top: 2rem;
+`;
+
+const CartListWrapper = styled.ul`
+  display: flex;
+  flex-direction: column;
+  gap: 0.2rem;
+  margin-bottom: 1rem;
+`;
+
+const CartItem = styled.li`
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(1, 1fr);
+  margin: 0.2rem;
+  padding: 1rem;
+`;
+
+const CartItemPic = styled.img`
+  object-fit: fill;
+  max-width: 100%;
+  max-height: 100%;
+  grid-column: 1/2;
+  grid-row: 1/2;
+  border: 1px solid #084d2c;
+`;
+
+const CartItemText = styled.p`
+  grid-column: 2/3;
+  grid-row: 1/2;
+  align-self: center;
+  padding: 0.8rem;
+  font-weight: bold;
+  font-size: 0.85rem;
+`;
+
+const CartItemCounter = styled.div`
+  grid-column: 2/3;
+  grid-row: 1/2;
+  align-self: end;
+`;
+
+const CartItemQuantityAndPrice = styled.div`
+  grid-column: 3/4;
+  grid-row: 1/2;
+  align-self: end;
+`;
+
+const CartItemSeparation = styled.hr`
+  border: 1px solid var(--color-green);
+  border-radius: 5px;
+`;
+
+const PlusButton = styled.button`
+  background-color: var(--color-light-gray);
+  border: 1px solid var(--color-black);
+  border-radius: 8px;
+  color: var(--color-black);
+  margin: 0.2rem;
+  padding: 0.2rem;
+`;
+
+const MinusButton = styled.button`
+  background-color: var(--color-light-gray);
+  border: 1px solid var(--color-black);
+  border-radius: 8px;
+  color: var(--color-black);
+  margin: 0.2rem;
+  padding: 0.25rem;
+`;
+
+const DeleteButton = styled.button`
+  background-color: var(--color-light-gray);
+  border: 1px solid var(--color-black);
+  color: var(--color-black);
+  margin: 0.2rem;
+  padding: 0.25rem;
+  position: absolute;
+  width: fit-content;
+  right: 1rem;
+`;
+
+const CoffeePriceBox = styled.div`
+  display: grid;
+  grid-template-columns: 2.5fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr;
+  margin: 0.2rem;
+  padding: 0.5rem;
+`;
+
+const CoffeePriceText = styled.div`
+  grid-column: 1/2;
+  grid-row: 1/2;
+  justify-self: start;
+`;
+
+const CoffeePrice = styled.div`
+  grid-column: 4/5;
+  grid-row: 1/2;
+  justify-self: end;
+`;
```

Comment:
Since you put text directly as child inside this element, I would rather use a p element than a div. It should work as well because both are block items.